src/pat/inject/inject.js

```diff
@@ -206,6 +207,14 @@ define([
                     cfg.$target.removeClass('is-dirty');
                 }
 
+                var $form = cfg.$target.parents('form')
+                if ($form.size() !== 0 && cfg.$target.data('initial-value') === undefined) {
+                    cfg.$target.data('initial-value', cfg.$target.html());
+                    $form.on('reset', function() {
+                        cfg.$target.html(cfg.$target.data('initial-value'))
+                    })
+                }
+
```

Comment:
Thanks for the explanation, makes sense. Please just present it in such a way that it's clear what's happening. I'd even go so far to put it in a separate method, such as `restoreOnFormReset`.